packages/editor-ui/src/stores/settings.store.ts

```diff
@@ -193,6 +197,14 @@ export const useSettingsStore = defineStore(STORES.SETTINGS, {
 		isDevRelease(): boolean {
 			return this.settings.releaseChannel === 'dev';
 		},
+		posthogSettings(): PosthogSettings {
+			const disableSessionRecording = !window.location.hostname.endsWith('n8n.cloud');
```

Comment:
wouldn't this now enable on the preview instance? I think we should revert back to the deployment type check instead.

Reply:
Didn't think of that. Let's go with the deployment type check